src/test/java/org/magicdgs/readtools/cmd/plugin/TrimmerPluginDescriptorUnitTest.java

```diff
@@ -24,6 +24,7 @@
 
 package org.magicdgs.readtools.cmd.plugin;
 
+import org.magicdgs.readtools.tools.trimming.TrimReadsTrimmerPluginArgumentCollection;
```

Comment:
Probably this class should implement a naive argument collection for testing (or a test-wise collection, depending on the behaviour to be tested), instead the `TrimReads` version. Now I don't have the time to do so, but a ticket should be open for that.

Reply:
Issue  #351